doc/apiv3-documentation.apib

```diff
@@ -2569,9 +2569,9 @@ Note that due to sharing this might be more than the versions *defined* by that
 | parentId         | Parent work package id                                 | Integer    | Must be an id of an existing and visible (for the current user) work package | READ / WRITE         |                                  |
 | startDate        |                                                        | Date       | must be equal or greater than the soonest possible start date                | READ / WRITE         |                                  |
 | dueDate          |                                                        | Date       | must be greater than or equal to the start date                              | READ / WRITE         |                                  |
-| estimatedTime    |                                                        | Duration   |                                                                              | READ                 |                                  |
+| estimatedTime    | Time a work package likely needs to be completed       | Duration   | Cannot be set for parent work packages                                       | READ / WRITE         |                                  |
 | spentTime        |                                                        | Duration   |                                                                              | READ                 | **Permission** view time entries |
-| percentageDone   |                                                        | Integer    | 0 <= x <= 100                                                                | READ                 |                                  |
+| percentageDone   | Amount of total completion for a work package          | Integer    | Cannot be set for parent work packages 0 <= x <= 100                         | READ / WRITE         |                                  |
```

Comment:
You should separate both constraints by a semicolon and possibly switch their positions... otherwise I could read that as a single constraint...

Proposal:
````
0 <= x <= 100; Cannot be set for parent work packages
````